deployment.yml

```
@@ -0,0 +1,55 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: event-be-deploy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: event-be
+  template:
+    metadata:
+      labels:
+        app: event-be
+    spec:
+      containers:
+        - name: event-be
+          image: lukianovaoleksandra/event_be:v1
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+          env:
```

As you can see, you have a lot of different variables, and there may be many more in your future projects. So, for better readability and usability, try to use configmaps and secrets. 
And instead of setting each variable separate, you can use something like that:
```suggestion
          envFrom:
            - configMapRef:
                name: db-config
            - secretRef:
                name: db-secret
```

---

Thanks for comment, done.